internal/providers/terraform/aws/rds_cluster.go

```diff
@@ -126,29 +147,25 @@ func NewRDSCluster(d *schema.ResourceData, u *schema.UsageData) *schema.Resource
 }
 
 func auroraStorageCostComponent(region string, u *schema.UsageData, databaseEngineStorageType *string) []*schema.CostComponent {
-	storageGB := decimal.Zero
-	if u != nil && u.Get("storage_gb").Exists() {
-		storageGB = decimal.NewFromFloat(u.Get("storage_gb").Float())
-	}
+	var storageGB, writeRequestsPerSecond, readRequestsPerSecond, monthlyIORequests *decimal.Decimal
 
-	writeRequestsPerSecond := decimal.Zero
-	if u != nil && u.Get("write_requests_per_sec").Exists() {
-		writeRequestsPerSecond = decimal.NewFromInt(u.Get("write_requests_per_sec").Int())
+	if u != nil && u.Get("storage_gb").Exists() {
+		storageGB = decimalPtr(decimal.NewFromFloat(u.Get("storage_gb").Float()))
 	}
 
-	readRequestsPerSecond := decimal.Zero
-	if u != nil && u.Get("read_requests_per_sec").Exists() {
-		readRequestsPerSecond = decimal.NewFromInt(u.Get("read_requests_per_sec").Int())
+	if u != nil && u.Get("write_requests_per_sec").Exists() && u.Get("read_requests_per_sec").Exists() {
+		writeRequestsPerSecond = decimalPtr(decimal.NewFromInt(u.Get("write_requests_per_sec").Int()))
+		readRequestsPerSecond = decimalPtr(decimal.NewFromInt(u.Get("read_requests_per_sec").Int()))
+		monthlyIORequestsD := calculateIORequests(*readRequestsPerSecond, *writeRequestsPerSecond)
+		monthlyIORequests = &monthlyIORequestsD
```

Comment:
monthlyIORequestsD variable is not necessary here
```suggestion
              monthlyIORequests = decimalPtr(calculateIORequests(*readRequestsPerSecond, *writeRequestsPerSecond))
```